Zotero/Scenes/Master/Collections/Views/CollectionsViewController.swift

```diff
@@ -70,6 +78,19 @@ final class CollectionsViewController: UICollectionViewController {
                       .disposed(by: self.disposeBag)
     }
 
+    override func viewIsAppearing(_ animated: Bool) {
+        super.viewIsAppearing(animated)
+
+        if !isSplit {
```

Comment:
This will add it even if in collapsed mode in iPad, but it will not remove it if it changes back to split mode. Probably it's ok though, no need to overthink it.

Reply:
Yeah I thought about it but it's pretty hidden and I at least added it to `viewIsAppearing` so that it's removed at some point. Though even if you pull to refresh in split mode it's not the end of the world. I felt like observing the split/collapsed state change and updating based on that was unnecessary.